.gitmodules

```
@@ -63,4 +63,8 @@
 	url = https://github.com/LuaCATS/tex-luatex.git
 [submodule "addons/slnunicode/module"]
 	path = addons/slnunicode/module
-	url = https://github.com/LuaCATS/slnunicode.git
\ No newline at end of file
+	url = https://github.com/LuaCATS/slnunicode.git
+[submodule ".\\addons\\xmake\\module"]
+	path = .\\addons\\xmake\\module
```

Please use UNIX paths
```suggestion
[submodule "addons/xmake/module"]
	path = addons/xmake/module
```

---

sure. This content is generated by running `git submodule`. let me update it manually